### ISSUE TITLE
Image URLs in markdown don't show up

### ISSUE THREAD

**Issue by zcysxy** [kisstkondoros/gutter-preview#91]
Only local images show the preview. The following types of images won't have a preview.

```markdown
![test](https://raw.githubusercontent.com/zcysxy/Figurebed/master/img/20200824154023.png)
```

```markdown
<image src="https://raw.githubusercontent.com/zcysxy/Figurebed/master/img/20200824154023.png">
```

**Comment by zcysxy** [comment]
I found the problem. Because I'm using a proxy to reach these images, and the extension works fine when I turn off the proxy (however then I'm blocked by the "wall"). So is there a way to make the extension use my system proxy? 🙏🙏🙏

**Comment by kisstkondoros** [comment]
Totally forgot about github issues in the past half year sorry.

Unfortunately this is not something which would be solvable via any vscode API at the moment, and managing something like this is out of scope for this extension.

Related vscode issue:
https://github.com/microsoft/vscode/issues/12588
